.github/workflows/build.yml

```diff
@@ -6,180 +6,158 @@ on:
       - 'dependabot/**'
   pull_request: ~
   release:
-    types: [created]
+    types: [ created ]
   schedule:
-    -
-      cron: "0 1 * * 6" # Run at 1am every Saturday
+    - cron: "0 1 * * 6" # Run at 1am every Saturday
   workflow_dispatch: ~
 
 jobs:
   tests:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
 
     name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
     strategy:
       fail-fast: false
       matrix:
-        php: ["8.0", "7.4"]
-        symfony: ["^4.4", "^5.2"]
-        sylius: ["~1.9.0", "~1.10.0", "~1.11.0"]
-        node: ["12.x"]
-        mysql: ["8.0"]
+        php: [8.0]
+        symfony: [^5.4, ^6.0]
+        sylius: [~1.11.0, ~1.12.0]
+        node: [^14.17.x]
+        mysql: [5.7, 8.0]
 
         exclude:
-          - sylius: ~1.9.0
-            php: 8.0
-
-          - sylius: ~1.10.0
-            symfony: 4.4
-
           - sylius: ~1.11.0
-            php: 7.4
-
+            symfony: ^6.0
     env:
       APP_ENV: test
       DATABASE_URL: "mysql://root:root@127.0.0.1/sylius?serverVersion=${{ matrix.mysql }}"
 
     steps:
-      -
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
-      -
-        name: Setup PHP
+      - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
           php-version: "${{ matrix.php }}"
           extensions: intl
           tools: symfony
           coverage: none
 
-      -
-        name: Setup Node
+      - name: Setup Node
         uses: actions/setup-node@v1
         with:
           node-version: "${{ matrix.node }}"
 
-      -
-        name: Shutdown default MySQL
+      - name: Shutdown default MySQL
         run: sudo service mysql stop
 
-      -
-        name: Setup MySQL
+      - name: Setup MySQL
         uses: mirromutth/mysql-action@v1.1
         with:
           mysql version: "${{ matrix.mysql }}"
           mysql root password: "root"
 
-      -
-        name: Output PHP version for Symfony CLI
+#      - name: Configure sysctl limits
```

Comment:
What's this? Why it's commented out?

Reply:
It was copied from other project, turns out unnecessary, i removed it in next commit